index.js

```diff
@@ -32,12 +34,23 @@ const main = async () => {
 					GITHUB_AUTH_TOKEN: token,
 				},
 			});
-			repoOSSF[repository] = output.slice(17).replace('\n', '');
+			const rateLimit = async () => {
+				const response = await octokit.request('GET /rate_limit');
+				return response;
+			};
+			const getRateLimit = await rateLimit();
+			console.log(getRateLimit.data.rate);
+			const writeOSSF = async () => {
+				repoOSSF[repository] = await output.slice(17).replace('\n', '');
```

Comment:
what's the magic 17 for?

also, this only replaces one newline; is that intended?

Reply:
It might be clearer to use the actual string with `.length` rather than 17 :-)

Sounds good on the newlines - probably some text fixtures would make this clearer also.

Reply:
I did not understand, how can we splice the output string with `.length`?